<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserSettingDialog</class>
 <widget class="QDialog" name="UserSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>191</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="TextLableLayout">
    <item>
     <layout class="QVBoxLayout" name="LabelLayout">
      <item>
       <widget class="QLabel" name="NameLabel">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="PeriodLabel">
        <property name="text">
         <string>Period</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="PaymentLabel">
        <property name="text">
         <string>Payment</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="TextLayout">
      <item>
       <widget class="QLineEdit" name="NameText"/>
      </item>
      <item>
       <widget class="QLineEdit" name="PeriodText"/>
      </item>
      <item>
       <widget class="QLineEdit" name="PaymentText"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>231</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="InfoLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Please enter the information
 of the new user and click 
 &quot;Add User&quot; button</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>124</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="AddButton">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Add User</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="RemoveButton">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Remove User</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="QuitButton">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="InfoLabel_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Please choose one user and click 
  &quot;Remove User&quot; button</string>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>tableView</zorder>
  <zorder>InfoLabel</zorder>
  <zorder>InfoLabel_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
